### PR TITLE
Updated InStore App documentation

### PR DESCRIPTION
**Pull request by nholzberger97**
Added POS API Helper and POS API configuration URL, added URL to download InStore App from App Center

- [ ] You've read the [Contributor Guide](https://github.com/fiskaltrust/.github/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/fiskaltrust/.github/blob/main/CODE_OF_CONDUCT.md).
- [ ] You've included unit or integration tests for your change, where applicable.
- [ ] You've included inline docs for your change, where applicable.
- [ ] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

{Summary of the changes}

## Description

{Detail}

Fixes #{issue number}

### CODE REVIEW

**Comment by nholzberger97** [comment]
@fiskaltrust/team-poscreator-experience please merge, many thanks.

File: doc/digital-receipt/Instore-App/digital-receipt-instore-app.md [hunk 1]
```diff
@@ -60,21 +60,32 @@ Please visit following link to see configuration steps for the master data:
...
-* SUNMI App Store
-* Google Play Store - soon
-* MAXSTORE (PAX) – soon
+
+* APK (via App Center) – (https://install.appcenter.ms/orgs/fiskaltrust/apps/in-store/distribution_groups/stable)
+* SUNMI App Store - Approved and available for: P2 PRO, P2, V1, V2, & V2 PRO
+* Google Play Store - coming soon
+* MAXSTORE (PAX) - Approved and available for: A35, A80 & A920Pro
 
 ### Pair InStore App
```
Comment (mijomilicevic) [inline doc/digital-receipt/Instore-App/digital-receipt-instore-app.md:91]: i would add an _<br / >_ for step 3 to have a linbreak after the screenshot (line 98)

Comment (mijomilicevic) [inline doc/digital-receipt/Instore-App/digital-receipt-instore-app.md:91]: i think it should be "You can pair multiple InStore App installationS with one CashBox." in line 98

**Review by mijomilicevic — CHANGES_REQUESTED** [review]

**Review by nholzberger97 — COMMENTED** [review]
added <br/> and missing s in row 98

**Review by mijomilicevic — APPROVED** [review]

**Review by mijomilicevic — APPROVED** [review]

**Pull request merged**